orchestra/bots/staffbot.py

```diff
@@ -107,6 +108,7 @@ def staff(self, task_id,
                     'text': error_msg
                 }])
 
+        close_open_staffbot_requests(task)
```

Comment:
these calls are good. we should also call close_open_staffbot_requests if a worker is manually assigned to a task (e.g., through project management or if they pick up a task from their dashboard). once you add that call, you can also clean up code that previously created a "fake" response from the manually assigned worker.

Reply:
(apologies if you did this and i didn't see it somewhere)

Reply:
It is already taken care of in `mark_worker_as_winner` which is called by `assign_task` and `reassign_task`. But you give me an idea of how to make `mark_worker_as_winner` a little cleaner.